app/helpers/sessions_helper.rb

```diff
@@ -15,6 +15,10 @@ def current_user
     end
   end
 
+  def logged_in?
+    !current_user.nil?
```

Comment:
present? thay thế nil? được k em

Reply:
Không được anh vì current_user nếu chưa đăng nhập thì nó sẽ nill mà nil.nil? sẽ trả về true còn nill.present? trả về false

Reply:
Oh, I understand your mindset. But I want to code that "current_user.present?"